cmd/debug-mobile/main.go

```diff
@@ -0,0 +1,18 @@
+package main
```

Comment:
I'm not sure about this, maybe `github.com/kubenav/kubenav/cmd/mobile/debug` would a better package path 🤔

Reply:
Suggested change implemented.